config_template/test_param_config_template.h

```diff
@@ -51,12 +51,9 @@
  */
 
 /**
- * @brief The IoT Thing name for the device for OTA test and MQTT test.
- * From https://docs.aws.amazon.com/iot/latest/developerguide/iot-thing-management.html,
- * MQTT_CLIENT_IDENTIFIER is different from IOT_THING_NAME. But we use IOT_THING_NAME
- * to be MQTT_CLIENT_IDENTIFIER here in MQTT Test and OTA E2E test.
+ * @brief The client identifier for MQTT test.
  *
- * #define IOT_THING_NAME  "PLACE_HOLDER"
+ * #define MQTT_TEST_CLIENT_IDENTIFIER    "PLACE_HOLDER"
```

Comment:
Is this ever actually different from IOT_THING_NAME?

Reply:
No, actually. 
MQTT_TEST_CLIENT_IDENTIFIER is used for MQTT test as client ID.
IOT_THING_NAME is used for OTA E2E test as thing name.
They are different, but they can be same string.

Reply:
More defination [here](https://docs.aws.amazon.com/iot/latest/developerguide/iot-thing-management.html).